src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,31 @@
+.catalog {
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fit, minmax($card-width-size, 1fr));
+
+  @media (max-width: 487px) {
```

Comment:
no need in`@media (max-width: 487px)` - you can write properties in `.catalog` directly